src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/scriptrunner/scriptrunner.py

```diff
@@ -81,8 +83,9 @@ def _main(settings, optlist, args):
         else:
             log.error("Error: No exception thrown, but XUnit results not created")
             test_executor.report_error(settings, failure_type="XUnitTestFailure")
+            return WORKITEM_FAILED
```

Comment:
You're right, it is logged. For reference, these are the return codes used from version 2.2 beta1 (we currently use beta2). http://xunit.github.io/docs/getting-started-desktop#console-return-codes